doc/express-reboot/Cisco_8000_Express_Reboot_HLD.md

```diff
@@ -0,0 +1,123 @@
+#
+# Sonic Express reboot HLD Spec
+
+## Overview
+
+The goal of Sonic express reboot is to be able to restart and upgrade SONiC software with sub-second data plane interruption. This is achieved by not initializing external phys, and only initializing NPU after receiving all configuration updates. This document covers updates needed in Sonic and SAI interface. 
```

Comment:
Can you create a section to describe the timelines on control plane and data plane version changes during this process, how the control traffic will be handled especially when control plane and data plane are not in the same version of images?

"all configuration updates", do you include operation data such as BGP routes learnt from remote side to be updated as well?

Reply:
@eddieruan-alibaba add a drawing that illustrates the interactions between v1/v2 sonic/sdk/npu during image upgrade. Please let me know if it is what you were looking for.